docs/guide/custom-functions.md

```diff
@@ -140,16 +220,83 @@ the calculations.
 // arguments here are displayed just to show the structure
 public hyper(ast, state) {
     return 'Hyperformula'.length;
+  };
+```
+
+### `runFunction()`
+
+To test your function's configuration, run the built-in `runFunction()` function.
+
+The `runFunction()` function:
+- Validates your function's [optional parameter](#optional-parameters) settings
+- Validates your [argument validation options](#argument-validation-options)
+- Checks if values returned by your function are in the right format
+
+```javascript
+import { FunctionPlugin } from 'hyperformula';
+
+export class CountHF extends FunctionPlugin {
+
+  public static implementedFunctions = {
+    'HYPER': {
+      method: 'hyper',
+      // set your optional parameters
+      arrayFunction: false,
+      // set your argument validation options
+      parameters: {
+        passSubtype: false
+      }
+    }
+  };
+
+  public hyper(ast, state) {
+    this.runFunction(HYPER())
+    
+    return 'Hyperformula'.length
```

Comment:
@izulin should we return `runFunction` results? I think it parses args, and vectorizes or sth the results

```suggestion
    return this.runFunction(() => 'Hyperformula'.length )
```

Reply:
@wojciechczerniak - updated: 1d1bada

Reply:
yes, but `runFunction` takes more args

Reply:
see example usage:

```
  public isodd(ast: ProcedureAst, state: InterpreterState): InterpreterValue {
    return this.runFunction(ast.args, state, this.metadata('ISODD'),
      (val) => (val % 2 === 1)
    )
  }
```